packages/eslint-config-base/index.js

```diff
@@ -21,7 +21,10 @@ module.exports = {
   settings: {
     'import/resolver': {
       node: {
-        extensions: ['.js', '.ts'],
+        extensions: ['.js', '.jsx', '.ts', '.tsx', '.mjs'],
```

Comment:
do yo need ts and tsx in here?

Reply:
not sure but the typescript config from the import plugin includes it...